data/data/bootstrap/files/usr/local/bin/wait-for-ha-api.sh

```diff
@@ -57,7 +57,7 @@ is_topology_ha() {
 
 ##
 ## for HA cluster, we mark the bootstrap process as complete when there
-## are at least two IP addresses available to the endpoints
```

Comment:
It's worth updating to mention here that the bootstrap instance is itself one of the endpoints. We don't want to tear down the bootstrap instance until there are enough permanent instances to tolerate 1 unavailable instance.